src/data/emacs-mode/agda-input.el

```diff
@@ -1,4 +1,4 @@
-;;; agda-input.el --- The Agda input method
+;;; agda-input.el --- The Agda input method -*- lexical-binding: t; -*-
```

Comment:
I guess you did not intend `-*- lexical-binding: t; -*-` to be part of the file's title.

Reply:
Thanks. I moved the file-local variable to the beginning of file.